### DISCUSSION TITLE
How to build and install ?

### DISCUSSION THREAD

**Discussion by vEonz121** [input-leap/input-leap#1510] [Q&A] [answered]
Is there any guidance on how to get this running?

There is literally no installation guide that has been documented for this repo.... I don't know how to install it, the gulf of execution is pretty big here...

**Comment by shymega** [comment, marked as answer]
There is a wiki, and I've configured a GH Actions workflow to automatically push from the PR repo: https://github.com/input-leap/input-leap/wiki

**Reply by miezebieze** [reply]
but [this part](https://github.com/input-leap/input-leap/wiki/Building-on-Linux#building-from-source) seems to be wrong. 
i tried to build and install it on Archlinux from source.
build works, install fails.
i found the correct command on the AUR. instead of the last two lines of that section (and page) should be:
> sudo cmake --install build

**Reply by shymega** [reply]
@miezebieze Thanks, fixed in latest wiki commits.

**Comment by sithlord48** [comment]
For The install part https://github.com/input-leap/input-leap/discussions/1793#discussioncomment-8577620
